crates/rolldown/tests/fixtures/function/format/iife/custom_indent/artifacts.snap

```diff
@@ -0,0 +1,38 @@
+---
+source: crates/rolldown_testing/src/integration_test.rs
+expression: snapshot
+input_file: crates/rolldown/tests/fixtures/function/format/iife/custom_indent
+---
+# Assets
+
+## main.mjs
+
+```js
+(function(exports) {
+
+   "use strict";
+
+   //#region main.js
+function test() {
+	console.log("Hello, world!");
```

Comment:
Here also need to implement Indet  for `SourcemapSource`, it also need to adjust token position, it's is complex at now. I think we could leave it at now before we have a good way.

Reply:
Yes, it is required also. Sorry for forgetting it :(.

Reply:
And actually `rollup` does not modify any of the inner indent.